compiler/rustc_feature/src/builtin_attrs.rs

```diff
@@ -26,6 +26,11 @@ const GATED_CFGS: &[GatedCfg] = &[
     (sym::target_thread_local, sym::cfg_target_thread_local, cfg_fn!(cfg_target_thread_local)),
     (sym::target_has_atomic, sym::cfg_target_has_atomic, cfg_fn!(cfg_target_has_atomic)),
     (sym::target_has_atomic_load_store, sym::cfg_target_has_atomic, cfg_fn!(cfg_target_has_atomic)),
+    (
+        sym::target_has_atomic_equal_alignment,
+        sym::cfg_target_has_atomic,
+        cfg_fn!(cfg_target_has_atomic),
+    ),
```

Comment:
Usually we have tests to ensure that these things are indeed rejected when used on stable, e.g. `feature-gate-cfg-target-has-atomic.rs`. Could you extend those tests with this flag as well, to be sure it is indeed nightly-only?

Reply:
Yes, will do!